transaction-spec/src/test/java/com/bloxbean/cardano/client/transaction/spec/MultiAssetSpecTest.java

```diff
@@ -177,4 +211,74 @@ void subtractMultiAssetListsWhenAdditionalMultiAssetInFirstList() {
 
         assertThat(result, equalTo(exList));
     }
+
+    @Test
+    void subtractMultiAssetListsWhenAdditionalAssetInSecondList() {
```

Comment:
@nemo83 Shouldn't this be called `AdditionalPolicyInSecondList`, since there's an extra policy_id3 in the second list?

Reply:
yes correct